api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -524,6 +525,8 @@ def mix(  # noqa: C901
                      :ref:`new-plunger-flow-rates`.
         :param aspirate_delay: How long to wait after each aspirate in the mix, in seconds.
         :param dispense_delay: How long to wait after each dispense in the mix, in seconds.
+        :param final_push_out: Optional ``push_out`` value. If not specified, it will emit the
+                               default non-zero amount.
```

Comment:
Maybe something like this, with a cross reference for the default values:
```suggestion
        :param final_push_out: How much to push out after the final mix repetition. The pipette will not push out after earlier repetitions. If not specified or ``None``, the pipette will push out the
                               default non-zero amount. See :ref:`push-out-dispense`.
```

Reply:
Yeah. Work with Ed on the wording. But this should explain that the default will be a positive pushout of some mystery amount.

Reply:
Hopefully not a mystery amount! We have defaults [documented](https://docs.opentrons.com/v2/basic_commands/liquids.html#push-out-after-dispense) now.